src/main/java/frc/robot/subsystems/drive/Drivetrain.java

```diff
@@ -48,11 +50,19 @@ public class Drivetrain extends SubsystemBase {
   private Timer m_timer;
   private double m_prevTime;
 
-  // Odometry class for tracking robot pose
-  SwerveDriveOdometry m_odometry;
```

Comment:
Why are we removing the odometry? Don't we need the odometry in addition to the estimator?

Reply:
Either way, we need odometry for path planning, so please add it back

Reply:
I don't think we need odometry, the path planner seems to only need pose2d which the estimator provides, its a wrapper around odometry.

Reply:
That should be fine then. Please also update the swerve drive pose estimator in periodic with the addVisionMeasurement from the photon pose estimator from the vision subsystem. You should incorporate this into a separate PR (it's not urgent).

Reply:
It's not. I checked. The first comment I wrote was a bit hasty.